translations/hu.json

```diff
@@ -0,0 +1,2984 @@
+        "message": [
+          "Ez a prémium kód nem érvényes."
+        ]
+      },
+      "15": {
+        "note": "Felemelve, amikor egy eseményt akarsz elindítani, de ezen a szerveren már van elindítva esemény.",
+        "message": [
+          "Már fut egy esemény ezen a szerveren.\n",
+          "Ha már véget ért, kitörölheted a `/event delete` paranccsal."
+        ]
+      },
+      "16": {
+        "note": "Felemelve, amikor egy prémium funkciót prémium nélkül akarsz használni.",
+        "message": [
+          "SkyKings Premium szükséges, hogy végrehajtsd ezt a parancsot.\n",
+          "További információért írd be a `/premium info` parancsot."
+        ]
+      },
+      "17": {
+        "note": "Felemelve, amikor a bot-nak nincsenek képküldési engedélyei.",
+        "message": [
+          "Nincs engedélyem, hogy üzenetet kűldjek a kijelölt csatornában."
+        ]
+      },
+      "18": {
+        "note": "Felemelve, amikor a játékos érvnek teljesülnie kéne, de nem teljesül.",
+        "message": [
+          "Mindenképp meg kell adnod a keresendő játékost, mivel nem vagy ellenőrizve."
+        ]
+      },
+      "19": {
+        "note": "Felemelve, amikor egy érvénytelen eseményt indítanál.",
+        "message": [
+          "**{event}** egy nem érvényes esemény."
+        ]
+      },
+      "20": {
+          "note": "Felemelve, amikor az események már be vannak kapcsolva.",
+          "message": [
+            "Ez az esemény már be van kapcsolva."
+          ]
+      },
+      "21": {
+          "note": "Felemelve, amikor az események már ki vannak kapcsolva.",
+          "message": [
+              "Ez az esemény már ki van kapcsolva."
+          ]
+      },
+      "22": {
+          "note": "Felemelve amikor egy érvénytelen időzítőt állítanál be. (skyblock events config)",
+          "message": [
+              "**{time}** egy érvénytelen idő. Csak **0**, **5**, **10**, vagy **30** lehetséges."
+          ]
+      },
+      "23": {
+          "note": "Felemelve, amikor egy figyelmeztetés már be van kapcsolva. (skyblock events config)",
+          "message": [
+              "Ez a figyelmeztetés már be van kapcsolva."
+          ]
+      },
+      "24": {
+          "note": "Felemelve, amikor egy figyelmeztetés már ki van kapcsolva. (skyblock events config)",
+          "message": [
+              "Ez a figyelmeztetés már ki van kapcsolva."
+          ]
+      },
+      "25": {
+          "note": "Felemelve, amikor egy játékosnak nincsenek lehozandó SkyBlock profiljai.",
+          "message": [
+              "**{name}**-nek nincsenek SkyBlock profiljai."
+          ]
+      }
+    },
+    "checkFailure": {
+      "0": {
+        "note": "Felemelve, amikor egy felhasználónak nincs prémiumja, de szükséges.",
+        "message": [
+          "A parancs használatához rendelkezned kell a SkyKings Premium-mal.\n",
+          "Több információért használd a `/premium info` parancsot."
```

Comment:
The whole command name should be translated, including top level
Should also be in backticks for `this effect`